docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,234 @@
+
+```
+kubectl --context=gce-asia-east1-a get services nginx
+```
+
+```
+kubectl --context=gce-asia-east1a get svc nginx
+NAME      CLUSTER-IP     EXTERNAL-IP      PORT(S)   AGE
+nginx     10.63.250.98   104.199.136.89   80/TCP    9m
+```
+
+The above assumes that you have a context named 'gce-asia-east1a' configured in your client for your cluster in that zone.  The name and namespace of the underlying services will automatically match those of the Federated Service that you created above (and if you happen to have had services of the same name and namespace already existing in any of those clusters, they will be automatically adopted by the Federation and updated to conform with the specification of your Federated Service - either way, the end result will be the same). 
+
+The status of your Federated Service will automatically reflect the real-time status of the underlying Kubernetes services, for example:
+
+```
+kubectl --context=federation-cluster describe services nginx
+```
+
+```
+Name:                   nginx
+Namespace:              default
+Labels:                 run=nginx
+Selector:               run=nginx
+Type:                   LoadBalancer
+IP:         
+LoadBalancer Ingress:   104.197.246.190, 130.211.57.243, 104.196.14.231, 104.199.136.89, ...
+Port:                   http    80/TCP
+Endpoints:              <none>
+Session Affinity:       None
+No events.
+```
+
+Note the the 'LoadBalancer Ingress' addresses of your Federated Service correspond with the 'LoadBalancer Ingress' addresses of all of the underlying Kubernetes services (once these have been allocated - this may take a few seconds).  For inter-cluster and inter-cloud-provider networking between service shards to work correctly, your services need to have an externally visible IP address.  Service Type: Loadbalancer is typically used for this.
+
+Note also what we have not yet provisioned any backend Pods to receive the network traffic directed to these addresses (i.e. 'Service Endpoints'), so the Federated Service does not yet consider these to be healthy service shards, and has accordingly not yet added their addresses to the DNS records for this Federated Service (more on this aspect later). 
+
+## Adding Backend Pods
+
+To render the underlying service shards healthy, we need to add backend Pods behind them.  This is currently done directly against the API endpoints of the underlying clusters (although in future the Federation server will be able to do all this for you with a single command, to save you the trouble).  For example, to create backend Pods in 13 underlying clusters:
+
+```
+for CLUSTER in asia-east1-c asia-east1-a asia-east1-b \
+                        europe-west1-d europe-west1-c europe-west1-b \
+                        us-central1-f us-central1-a us-central1-b us-central1-c \
+						us-east1-d us-east1-c us-east1-b
+do
+  kubectl --context=$CLUSTER run nginx --image=nginx:1.11.1-alpine --port=80
```

Comment:
I guess `kubectl run` automatically adds the `run=nginx` label?  If so, that's nice and ties the example together, but you might wish to say so explicitly.